src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -20,10 +20,15 @@
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.person.Address;
+import seedu.address.model.person.Birthdate;
 import seedu.address.model.person.Email;
+import seedu.address.model.person.Gender;
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Person;
 import seedu.address.model.person.Phone;
+import seedu.address.model.person.Race;
+import seedu.address.model.person.Religion;
+import seedu.address.model.person.Survey;
 import seedu.address.model.tag.Tag;
 
 /**
```

Comment:
Should probably check MESSAGE_USAGE too

Reply:
Will leave for the person doing the changes in the edit command.